src/main/frontend/handler/extract.cljs

```diff
@@ -146,7 +146,7 @@
           pages (remove nil? pages)
           pages (map (fn [page] (assoc page :block/uuid (db/new-block-id))) pages)
           blocks (->> (remove nil? blocks)
-                      (map (fn [b] (dissoc b :block/title :block/body :block/level))))]
+                      (map (fn [b] (dissoc b :block/title :block/body))))]
```

Comment:
@tiensonqin I saw this was recently changed in #3362. I added back :block/level in order for tests to pass - https://github.com/logseq/logseq/pull/3851/commits/170391024a698dfc9cff889174d1dd3cc5b683f7 . If this isn't desired I can remove levels from test assertions

Reply:
@logseq-cldwalker `:block/level` is included in the db schema right now, but it doesn't have to, because a block can be located by both `:block/left` and `:block/parent`, so its level can be computed then. I think that's the reason why I removed it in #3362.
We can address this later with another PR.